### DISCUSSION TITLE
What does "beta" mean in terms of react-aria hooks?

### DISCUSSION THREAD

**Discussion by cssinate** [adobe/react-spectrum#5758] [Q&A]
All of the `useColorX` hooks are marked as "beta". What does this mean in practical terms? What risks am I assuming by using them in production?

**Comment by reidbarber** [comment]
Check out this section of the docs on pre-release stages: https://react-spectrum.adobe.com/Support.html#pre-releases

You can also search through the open issues and see if any of them would affect your use case.
